docs/rationale.md

```diff
@@ -116,3 +116,13 @@ completion in Spring 2023, with the [migration of Ahrefs frontend codebase to
 Melange](https://tech.ahrefs.com/ahrefs-is-now-built-with-melange-b14f5ec56df4)
 and the new public releases that support it: version 3.8 of Dune and 1.0 of
 Melange.
+
+## Looking forward
+
+While reaching v1.0 marks a major milestone for Melange, it is only the 
+beginning of the journey. The Melange team remains committed to
+continuously improving Melange, ensuring it remains a robust and efficient
+tool for OCaml developers targeting the JavaScript platform.
+The [Q2 2023
```

Comment:
@anmonteiro @mooreryan In a couple of weeks we will be in Q3 already, so this will be obsolete. Is there a way we can adapt the new "Looking forward" section to avoid quick obsolescence? Maybe we could add a `/roadmap` section in the documentation site and link to it instead of the google doc?

Reply:
Sorry for not giving you a chance to review; you bring up a great point.

I like the idea of having a roadmap page / section where we list all the documents we've written as part of planning out the melange roadmap. That'd give a good historical view of what has been worked on, along with promoting transparency.

Reply:
np 👍  created https://github.com/melange-re/melange-re.github.io/issues/82 for now.